lib/new_relic/agent/instrumentation/sunspot.rb

```diff
@@ -11,8 +11,8 @@
 
   executes do
     ::NewRelic::Agent.logger.info 'Installing Rails Sunspot instrumentation'
-    deprecation_msg = 'The instrumentation for Sunspot is deprecated.' \
-      ' It will be removed in version 9.0.0.' \
```

Comment:
My guess is that this trailing slash would cause issues should it ever get executed. Perhaps we've broken support for Sunspot earlier than expected?

Reply:
Nice catch! I guess that makes it two public-facing changes this release 😉